code/modules/power/cable.dm

```diff
@@ -439,7 +439,7 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list(new/datum/stack_recipe("cable restrain
 	return(OXYLOSS)
 
 /obj/item/stack/cable_coil/proc/check_menu(mob/living/user)
-	if(!istype(user))
+	if(!istype(user) || !iscyborg(user))

```

Comment:
you sure about this one, chief?

won't this make this return false for anyone who ISN'T a cyborg?

Reply:
Humm yea your right I thought being human was already checked.

Reply:
```
	if(!iscyborg(user) && !ishuman(user))
		return FALSE
```
This better? ishuman is a catch all for people who can work on wires?

Reply:
Wish it was like machinery where these checks are done before attack hand